### DISCUSSION TITLE
Power Consumption - Showcase

### DISCUSSION THREAD

**Discussion by avarga** [inet-framework/inet#600] [Showcases]
Discuss the "Power Consumption" showcase here. Ideas and recommendations are welcome.

**Comment by majedaboumosab** [comment]
Hi
How can I calculate the power consumption as in this example in showcases 
The sum of the power consumption of the four sensors and the gateway (values in Joules.)

the are two parameters in anf file one is power consumption and another is residualEnergyCapacity but it is negative values, 
Another thing there are mean, max an min 

and also , how i invert the negative values?
